File: Function/5.py
input_id = input("아이디를 입력해주세요: ")

def login(id):
    members = ['egoing', 'k8805']
    members.append('leezche')

    for member in members:
        if member == input_id:
            return True
    return False

if login(input_id):
    print('Hello, '+input_id+'!')
else:
    print('Who are you?')

File: Function/5.rb
print("아이디를 입력해주세요: ")
input_id = gets.chomp()

def login(id)
  members = ['egoing', 'k8805']
  members << 'leezche'

  for member in members do
    if member == id
      return true
    end
  end
  return false
  
end

if(login(input_id))
  print('Hello, '+input_id+'!')
else
  print('Who are you?')
end
